cognite/client/_api/sequences.py

```diff
@@ -69,7 +71,8 @@ class SequencesAPI(APIClient):
 
     def __init__(self, config: ClientConfig, api_version: str | None, cognite_client: CogniteClient) -> None:
         super().__init__(config, api_version, cognite_client)
-        self.data = SequencesDataAPI(config, api_version, cognite_client)
+        self.rows = SequencesRowsAPI(config, api_version, cognite_client)
```

Comment:
Why this change? Seems unnecessary. No longer mirrors the time series client, and also devices from the API where it's sequences/data.